prime-router/docs/getting-started/fhir-functions.md

```diff
@@ -1,7 +1,68 @@
 # FHIR Functions
+FHIR functions are methods that can be run on a bundle in order to extract data from it (ex. retrieve an age or telephone area code)
+Currently, there are two places where FHIR functions are created. Both are in a file called CustomFhirPathFunctions and
+both extend an interface called FhirPathFunctions. There are two places for these functions because some
+are internal functions, meaning that they can only be called from within the code rather than from the command line.
```

Comment:
are internal functions, meaning that they can only be called from within the code rather than from the command line **via the [fhir path tool](internal link here).**